File: Java/ObjectsAndClasses.java
package Java;

public class ObjectsAndClasses {
    // Class definition
    static class Person {
        String name;
        int age;

        // Constructor
        Person(String name, int age) {
            this.name = name;
            this.age = age;
        }
    }

    public static void main(String[] args) {
        // Object instantiation
        Person person = new Person("John", 25);

        // Accessing object properties
        System.out.println("Name: " + person.name);
        System.out.println("Age: " + person.age);
    }
}

File: C++/ObjectsAndClasses.cpp
#include <iostream>

// Class definition
class Person {
public:
    std::string name;
    int age;
};

int main() {
    // Object instantiation
    Person person;
    person.name = "John";
    person.age = 25;

    // Accessing object properties
    std::cout << "Name: " << person.name << std::endl;
    std::cout << "Age: " << person.age << std::endl;

    return 0;
}